css/selectors/-webkit-scrollbar.json

```
@@ -36,7 +36,8 @@
               "version_added": "4"
             },
             "safari_ios": {
-              "version_added": "3"
+              "version_added": "3",
+              "notes": "As of Safari for iOS 13 only `display: none` will work, any other styles have no effect."
```

Some style suggestions, for consistency with the way we write notes elsewhere (mostly, to be more concise, use HTML, and avoid a comma splice).

Also, a note in case anyone looks this up in the future: we didn't go with multiple support statements with `partial_implementation` because this is non-standard feature. It's unknowable what would constitute partial implementation, so we can only note the changes in behavior.

```suggestion
              "notes": "From Safari 13, only <code>display: none</code> works with this pseudo-element. Other styles have no effect."
```